float32 xAxisRotation
float32 yAxisRotation
float32 zAxisRotation

float32 xAxisVelocity
float32 yAxisVelocity
float32 zAxisVelocity

float32 xAxisAcceleration
float32 yAxisAcceleration
float32 zAxisAcceleration

float32[] rotationMatrix
